components/sensor/ld2410.rst

```diff
@@ -114,68 +275,49 @@ and binary sensors.
       - 30
       - 15
 
-Sensor
+Button
 ------
 
-The ``ld2410`` sensor allows you to use your :doc:`ld2410` to perform different
-measurements.
+The ``ld2410`` button allows you to perfrom actions on your :doc:`ld2410`.
 
 .. code-block:: yaml
 
-    sensor:
+    button:
       - platform: ld2410
-        moving_distance:
-          name : Moving Distance
-        still_distance:
-          name: Still Distance
-        moving_energy:
-          name: Move Energy
-        still_energy:
-          name: Still Energy
-        detection_distance:
-          name: Detection Distance
+        factory_reset:
+          name: "factory reset"
+        restart:
+          name: "restart"
+        query_params:
+          name: query params
 
 Configuration variables:
 ************************
 
-- **moving_distance** (*Optional*): Distance of detected moving target.
-  All options from :ref:`Sensor <config-sensor>`.
-- **still_distance** (*Optional*): Distance of detected still target.
-  All options from :ref:`Sensor <config-sensor>`.
-- **moving_energy** (*Optional*): Energy for moving target.
-  All options from :ref:`Sensor <config-sensor>`.
-- **still_energy** (*Optional*): Energy for still target.
-  All options from :ref:`Sensor <config-sensor>`.
-- **detection_distance** (*Optional*): Distance in cm of target.
-  All options from :ref:`Sensor <config-sensor>`.
+- **factory_reset** (*Optional*): Perfrom factory reset.
```

Comment:
Even expression 'factory_reset' is clear, we could add precision about exact behavior as in official doc : 
'This command is used to restore all configuration values to their original values, and the
configuration values will take effect **after restarting the module**.'

Reply:
factory reset here actually performs the restart already so I will omit that part

Reply:
done